### ISSUE TITLE
rxjs-tslint breaks tslint

### ISSUE THREAD

**Issue by NagRock** [ReactiveX/rxjs-tslint#44]
When I install `rxjs-tslint` with `yarn` every thing works fine. But when I remove `node_modules` and use again `yarn` to install all dependencies I get many errors like this one:

```
TypeError: Cannot read property 'length' of undefined
    at cb (C:\workspaces\angular-components\node_modules\tslint\lib\rules\noDuplicateVariableRule.js:79:66)
    at visitNodes (C:\workspaces\angular-components\node_modules\typescript\lib\typescript.js:13272:30)
    at Object.forEachChild (C:\workspaces\angular-components\node_modules\typescript\lib\typescript.js:13479:24)
    at NoDuplicateVariableWalker.walk (C:\workspaces\angular-components\node_modules\tslint\lib\rules\noDuplicateVariableRule.js:86:19)
    at Rule.AbstractRule.applyWithWalker (C:\workspaces\angular-components\node_modules\tslint\lib\language\rule\abstractRule.js:31:16)
    at Rule.apply (C:\workspaces\angular-components\node_modules\tslint\lib\rules\noDuplicateVariableRule.js:34:21)
    at Linter.applyRule (C:\workspaces\angular-components\node_modules\tslint\lib\linter.js:197:29)
    at C:\workspaces\angular-components\node_modules\tslint\lib\linter.js:139:85
    at Object.flatMap (C:\workspaces\angular-components\node_modules\tslint\lib\utils.js:151:29)
    at Linter.getAllFailures (C:\workspaces\angular-components\node_modules\tslint\lib\linter.js:139:32)
```

I don't need to add any rule to `tslint.json`, just add `rxjs-tslint` as dependency and install all dependencies depending on `yarn.lock` file.

Don't know if this is related to: https://github.com/palantir/tslint/issues/3711#issuecomment-389516346

**Comment by PsychicCat** [comment]
Seems to me that tslint 5.10 is causing the error.

**Comment by NagRock** [comment]
@PsychicCat I've created PR that should fix this issue. But need to wait to be merged. #45

**Comment by mgechev** [comment]
Published under 0.1.5.

**Comment by NagRock** [comment]
@mgechev thanks!
